spec/system/log_ins_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'LogIns', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:user) { FactoryBot.create(:user) }
```

Comment:
```suggestion
  let(:user) { create(:user) }
```
Check if it works like this